### DISCUSSION TITLE
Unable to start Seq in local machine

### DISCUSSION THREAD

**Discussion by joinsukesh** [datalust/seq-tickets#1806] [General]
My Seq portal was working fine.

I restarted my system, launched the Seq Service Administration dialog and it shows this.

![image](https://user-images.githubusercontent.com/24619393/228251696-a2f5c8a1-118e-4b13-a66f-a41ed4abef2a.png)

![image](https://user-images.githubusercontent.com/24619393/228251812-45827be2-baf9-407e-837e-e4851e585578.png)

**Comment by joinsukesh** [comment]
I closed the Windows Services panel, ran the Seq Service Administration. And it's working now!!!
Any thoughts, why it occurred in the first place?

**Reply by KodrAus** [reply]
Hi @sukesh-y :wave: The Service Administration dialog is really only needed during installation. Once you've got a Windows Service on your machine that should be all you need to keep Seq running. You can also interact with Seq [through `seq` on the commandline](https://docs.datalust.co/docs/server-command-line) which lets you start and stop that Windows service on-demand.

**Comment by todthomson** [comment]
My best guess is that having the Services window open while the Seq installer was trying to add/remove the Seq service (especially if you had the Service itself open in the background?) might have locked it so that it couldn't be updated by our script.
